src/internal/connector/onedrive/collection.go

```diff
@@ -410,7 +410,12 @@ func (oc *Collection) populateItems(ctx context.Context, errs *fault.Bus) {
 
 					// check for errors following retries
 					if err != nil {
-						logger.Ctx(ctx).With("error", err.Error()).Error("downloading item")
+						if item.GetMalware() != nil {
+							logger.Ctx(ctx).With("error", err.Error(), "malware", true).Error("downloading item")
```

Comment:
Didn't wanna add the malware key if it was not explicit present, but let me know if we should always log it.

Reply:
Could be useful to have this knowledge outside of errors.  I'd suggest adding it in the `clues.Add()` on 378 (whether or not the item is malware).